src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -151,6 +149,8 @@ object Task {
 
     def containerName(taskId: String): Option[String] = {
       taskId match {
+        case ResidentTaskIdWithInstanceIdRegex(runSpecId, prefix, uuid, container, attempt) =>
```

Comment:
`(_, _, _, container, _)`?

Reply:
It is less-readable. I intentionally kept it the way it was and still is.